contract/population-types.go

```diff
@@ -0,0 +1,13 @@
+package contract
+
+type PopulationTypes struct {
+	Items []PopulationType `json:"items"`
+}
+
+func NewPopulationTypes(names []string) *PopulationTypes {
+	items := make([]PopulationType, len(names))
+	for i, name := range names {
+		items[i] = *NewPopulationType(name)
+	}
+	return &PopulationTypes{Items: items}
```

Comment:
Do we need to return a pointer?
Could we just return a copy `PopulationTypes`?
```suggestion
	return PopulationTypes{Items: items}
```

Reply:
ok that seems simpler